Test for optForce

tpfau opened the pull request:
A test function for optForce and a small update to the FVA functionality.
optForceFVA now uses fluxVariability (can thus benefit from all updates to that function).

updated printLevel Setting for solveCobra calls in optForce function to the same as verbose (i.e. if verbose is 0 the printlevel is also 0).

Updated the default verbose setting of optForce to fit to documentation. 

Fixed a small bug in the optForce tutorial.


**I hereby confirm that I have:**

- [X] Tested my code on my own machine
- [X] Followed the guidelines in the [Contributing Guide](https://opencobra.github.io/cobratoolbox/docs/contributing.html)
- [X] Selected `develop` as a target branch (top left drop-down menu)

*(Note: You may replace [] with [X] to check the box)*

laurentheirendt commented:
The error on Windows is:
```
15:17:22 ================================================================================
15:17:22 Error occurred in testOptForce/testOptForce and it did not run to completion.
15:17:22 
15:17:22     ---------
15:17:22     Error ID:
15:17:22     ---------
15:17:22     'MATLAB:xlswrite:dlmwrite'
15:17:22 
15:17:22     --------------
15:17:22     Error Details:
15:17:22     --------------
15:17:22     Error using xlswrite (line 187)
15:17:22     An error occurred on data export in CSV format.
15:17:22     
15:17:22     Error in findMustL (line 317)
15:17:22             xlswrite([outputFileName '_Info'], Info);
15:17:22     
15:17:22     Error in testOptForce (line 59)
15:17:22             [mustLSet, pos_mustL] = findMustL(model, minFluxesW, maxFluxesW, 'constrOpt', constrOpt, ...
15:17:22     
15:17:22     Caused by:
15:17:22         Error using dlmwrite (line 112)
15:17:22         The input cell array cannot be converted to a matrix.
15:17:22 ================================================================================
```
May you have a look at that? Thanks 👍

tpfau commented:
Well, I guess that the test system does not have excel installed. So the matlab native xlswrite operations fail.
I changed this to xlwrite (anyways already part of the toolbox) and removed the windows only flag.
